cumulus/xcm/xcm-emulator/src/lib.rs

```diff
@@ -599,6 +600,7 @@ macro_rules! decl_test_parachains {
 					LocationToAccountId: $location_to_account:path,
 					ParachainInfo: $parachain_info:path,
 					MessageOrigin: $message_origin:path,
+					DigestProvider: $digest_provider:ty,
```

Comment:
Could be an optional argument?

Reply:
yes, let me try this

Reply:
made it optional by adding a new rule to the `decl_test_parachains` macro that puts the default digest provider () if the optional argument is not set or the provided one if it is set. 

I removed all digest provider arguments from the parachain tests except for the assethub ones, just to make sure we dont break it

Reply:
@girazoki you forgot  to push :D

Reply:
yeah haha, now I did I think

Reply:
I will remove the inclusion of the convert trait too in all the test runtimes